app/src/sharedTest/java/org/oppia/app/parser/HtmlParserTest.kt

```diff
@@ -99,36 +99,54 @@ class HtmlParserTest {
 
   @Test
   fun testHtmlContent_handleCustomOppiaTags_parsedHtmlDisplaysStyledText() {
-    val textView = activityTestRule.activity.findViewById(R.id.test_html_content_text_view) as TextView
+    val textView = activityTestRule.activity.findViewById(
+      R.id.test_html_content_text_view
+    ) as TextView
     val htmlParser = htmlParserFactory.create(
-      resourceBucketName, /* entityType= */ "", /* entityId= */ "", /* imageCenterAlign= */ true
+      resourceBucketName,
+      /* entityType= */ "",
+      /* entityId= */ "",
+      /* imageCenterAlign= */ true
     )
     val htmlResult: Spannable = htmlParser.parseOppiaHtml(
-      "\u003cp\u003e\"Let's try one last question,\" said Mr. Baker. \"Here's a pineapple cake cut into pieces.\"\u003c/p\u003e\u003coppia-noninteractive-image " +
-          "alt-with-value=\"\u0026amp;quot;Pineapple" +
-          " cake with 7/9 having cherries.\u0026amp;quot;\" caption-with-value=\"\u0026amp;quot;\u0026amp;quot;\" filepath-with-value=\"\u0026amp;quot;" +
-          "pineapple_cake_height_479_width_480.png\u0026amp;quot;\"\u003e\u003c/" +
-          "oppia-noninteractive-image\u003e\u003cp\u003e\u00a0\u003c/p\u003e\u003cp\u003e\u003cstrong\u003eQuestion 6\u003c/strong\u003e: What " +
-          "fraction of the cake has big red cherries in the pineapple slices?\u003c/p\u003e",
+      "\u003cp\u003e\"Let's try one last question,\" said Mr. Baker. \"Here's a " +
+        "pineapple cake cut into pieces.\"\u003c/p\u003e\u003coppia-noninteractive-image " +
+        "alt-with-value=\"\u0026amp;quot;Pineapple" +
+        " cake with 7/9 having cherries.\u0026amp;quot;\" caption-with-value=\"\u0026amp;quot;" +
+        "\u0026amp;quot;\" filepath-with-value=\"\u0026amp;quot;" +
+        "pineapple_cake_height_479_width_480.png\u0026amp;quot;\"\u003e\u003c/" +
+        "oppia-noninteractive-image\u003e\u003cp\u003e\u00a0\u003c/p\u003e\u003cp" +
+        "\u003e\u003cstrong\u003eQuestion 6\u003c/strong\u003e: What " +
+        "fraction of the cake has big red cherries in the pineapple slices?\u003c/p\u003e",
```

Comment:
Thank you for doing this :)

Reply:
You're Welcome :)

Reply:
Resolving this on behalf of @vinitamurthi